packages/esm-service-queues-app/src/queue-table-metrics-extensions/total-patients-in-queue.extension.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import QueueTableMetricsCard from '../queue-table/queue-table-metrics-card.component';
+import { useQueueEntriesMetrics } from '../hooks/useQueueEntries';
+
+interface TotalPatientsInQueueMetricsProps {
+  queueUuid: string;
+  statusUuid: string;
+}
+
+export default function TotalPatientsInQueueMetrics({ queueUuid, statusUuid }: TotalPatientsInQueueMetricsProps) {
+  const { t } = useTranslation();
+  const { count } = useQueueEntriesMetrics({
+    queue: queueUuid,
+    isEnded: false,
+  });
+
+  return (
+    <QueueTableMetricsCard
+      value={count}
+      headerLabel={t('totalPatients', 'Total Patients')}
+      linkTo={`\${openmrsSpaBase}/home/service-queues/queue-table-by-status/${queueUuid}`}
+    />
+  );
+}
```

Comment:
You shouldn't need this class.  Just change the `StatusQueueMetrics` component -> `QueueMetrics` component and make the `status` prop optional.  Then render the appropriate metric based on queue+status or just queue.